app/models/reports/repository.rb

```diff
@@ -63,42 +76,46 @@ def self.smart_memoize(method)
       end
     end
 
+    alias_method :adjusted_patient_counts_without_ltfu, :adjusted_patient_counts
```

Comment:
@rsanheim We should just call this `adjusted_patients_counts`, since it is the default now. The other one already implies that it is the `with_ltfu` version.